tests/foreman/ui/test_contentview.py

```diff
@@ -4334,3 +4336,71 @@ def test_positive_greedy_dep_solving_with_multiversion_packages(session, module_
             'name = {}'.format(property_name),
             conserve_param_value
         )
+
+
+@tier2
+def test_positive_depsolve_with_module_errata(session, module_org):
```

Comment:
I have added test https://github.com/SatelliteQE/robottelo/pull/7339/files#diff-03af7637f7d6dfabee1a2de7557ebb9bR950, please check once we are not repeating the same.

Reply:
These are very similar but I think your is more extensive in basic functionary of filtering on modular errata on api and ui.  Mine is more UI focus that is specific to a modular errata when filtering and depsolving.  It should still be a valid test case right?

Reply:
These are very similar but I think your is more extensive in basic functionary of filtering on modular errata on api and ui. Mine is more UI focus that is specific to a modular errata when filtering and depsolving. It should still be a valid test case right?